app/src/main/java/com/asfoundation/wallet/wallet_validation/poa/PoaWalletValidationActivity.kt

```diff
@@ -6,10 +6,12 @@ import android.os.Bundle
 import android.os.PersistableBundle
```

Comment:
Change the name of the class to IabWalletValidationActivity

Reply:
I can't be Iab since it's also used in the PoA.
Could it be DialogWalletValidationActivity?